sdk/python/kfp/v2/compiler/compiler.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google LLC
```

Comment:
I don't think we should update this date, since usually it marks the date from which this file exists, right?

Reply:
You're right. Reverted.